src/main.tsx

```diff
@@ -86,6 +87,10 @@ const router = createBrowserRouter(
           path: PageRoute.Export,
           element: FormPageHoC(ExportPage, {}),
         },
+        {
+          path: PageRoute.VisualisationExample,
+          element: <VisualisationExamplePage />,
```

Comment:
I have created a new component for the page which is just a wrapper around the Visualisation component with paths to the assets files. I like having it as a component in pages/ for consistency but we could also keep it simple and use the Visualisation component here. i.e.:
{
  path: PageRoute.VisualisationExample,
  element: (
    <Visualisation
       developmentPlanFilepath={"assets/development-plan.csv"}
       timetableEventsFilepath={"assets/timetable.csv"}
     />
 ),
 },

Reply:
No strong opinions on this :)

Reply:
could this be lazy loaded? just to keep in line with [this](https://github.com/digital-land/local-plans-timetable/pull/83) pr

Reply:
Done! Thanks George :)